tests/unit/test_webui_lazy_imports.py

```
@@ -0,0 +1,215 @@
+    monkeypatch.setitem(sys.modules, schemas_mod.__name__, schemas_mod)
+    monkeypatch.setitem(sys.modules, validator_mod.__name__, validator_mod)
+
+    import parakeet_rocm.webui.validation as validation
+
+    assert validation.TranscriptionConfig.__name__ == "TranscriptionConfig"
+    assert validation.FileUploadConfig.__name__ == "FileUploadConfig"
+    assert validation.validate_audio_file(object())
+    assert validation.validate_output_directory(object())
+
+    with pytest.raises(AttributeError):
+        getattr(validation, "Missing")
```

Ruff (0.14.13)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)